tip/tip.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import re
+from sys import argv, stdout, exit as sys_exit
+from os import environ, getenv
+from glob import glob
+from pathlib import Path
+from github import Github
+from github import GithubException
+
+print("· Get list of artifacts to be uploaded")
+
+args = []
+files = []
+
+if "INPUT_FILES" in environ:
+    args = environ["INPUT_FILES"].split()
+
+if len(argv) > 1:
+    args = args + argv[1:]
+
+if len(args) == 1 and args[0] == "none":
+    files = []
+    print("! Skipping 'files' because it's set to 'none")
+else:
+    if len(args) == 0:
+        stdout.flush()
+        raise (Exception("Glob patterns need to be provided as positional arguments or through envvar 'INPUT_FILES'!"))
+
+    for item in args:
+        items = [fname for fname in glob(item, recursive=True) if not Path(fname).is_dir()]
+        print(f"  glob({item!s}):")
+        for fname in items:
+            if Path(fname).stat().st_size == 0:
+                print(f"  - ! Skipping empty file {fname!s}")
+                continue
+            print(f"  - {fname!s}")
+            files += [fname]
```

Comment:
Use `files.append(fname)` instead of `+= [...]`

Reply:
Done.